shacl12-sparql/index.html

```diff
@@ -1315,6 +1317,79 @@ <h3>Select Expressions</h3>
 					</div>
 				</aside>
 			</section>
+
+			<section id="SPARQLExprExpression">
+				<h3>SPARQL Expr Expressions</h3>
+				<p>
+					A <a>node expression</a> that has a <a>value</a> for <code>sh:sparqlExpr</code> is called a <dfn>SPARQL expr expression</dfn> with the <a>function name</a>
```

Comment:
`sh:sparqlExpr` is not in `shacl.ttl`.  Was that intentional?

Reply:
It is, in sh:SPARQLExprExpression-sparqlExpr which is a sh:Parameter declaration. From a SHACL perspective there is no need to declare it globally, if you mean that, as it's only used as predicate.